nistats/first_level_model.py

```diff
@@ -464,6 +463,7 @@ def fit(self, run_imgs, events=None, confounds=None,
                 sys.stderr.write('Starting masker computation \r')
 
             Y = self.masker_.transform(run_img)
+            del run_img
```

Comment:
Add a comment (your audience is someone new to the project) why this is being done.

Reply:
Sure, although I see that deleting unused variables happens throughout the code base (for example in `run_glm`). So this doesn't seem any different from any other instance of memory saving practices throughout nistats.